CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 # Changelog
 Changelog for NeoFS Contract
 
+## [Unreleased]
```

Comment:
lets add diff link for this

0.17.0 link is also missing btw

Reply:
Yeah, I've noticed. Though to create another PR, but hey, let's have it here.